package.json

```diff
@@ -56,7 +56,7 @@
     "@typescript-eslint/eslint-plugin": "4.32.0",
     "@typescript-eslint/parser": "4.32.0",
     "bats": "1.4.1",
-    "cypress": "6.5.0",
+    "cypress": "7.6.0",
```

Comment:
@mudit94 can you please also update the `package-lock.json` ?

Reply:
I see its updated